.env.example

```diff
@@ -81,6 +81,7 @@ AWS_SECRET_ACCESS_KEY=
 # AWS_S3_CUSTOM_DOMAIN=None       # "example-bucket-name.s3.fr-par.scw.cloud"
 # AWS_S3_REGION_NAME=None         # "fr-par"
 # AWS_S3_ENDPOINT_URL=None        # "https://s3.fr-par.scw.cloud"
+# S3_ENDPOINT_URL=None            # same as AWS_S3_ENDPOINT_URL - needed for non-AWS for user exports
```

Comment:
Couldn't it be duplicated in the code itself and not by a user if it has the same value? Leaving this in the configuration could potentially lead to more misconfiguration problems.

Reply:
I certainly agree it would be cleaner but I'm not sure how to do it. [s3-tar](https://pypi.org/project/s3-tar/) doesn't provide a way to configure credentials, it just picks them up from environment variables, and it doesn't seem to be possible to make one env value equal another within a `.env` file.

Reply:
> needed for non-AWS for user exports

Do you mean [this](https://github.com/xtream1101/s3-tar/blob/master/s3_tar/utils.py#L25-L29)?

I'm not a Python developer at all and I do not know best practices here, but I have an idea.

What if Bookwyrm will set this in `os.environ` when reading it's settings? Of course it would probably be a good idea to also set it with [`os.putenv()`](https://docs.python.org/3/library/os.html#os.putenv) just to be consistent.

`s3-tar` reads `os.environ` only when `_create_s3_client()` is called and when it's called, Bookwyrm would've already set that.

I understand that I'm suggesting a dirty hack, but considering that `s3-tar` is written in a such way that it doesn't allow to override this value, for me personally using a dirty hack in the environment I as a developer of a product control is better than leaving the hack to the end-user especially if I'm completely sure that both variables should have the SAME value and user shouldn't use different values. This way I'm not giving my user an opportunity to mess things up by changing one and forgetting to change another or not setting one of them at all for example.

I don't like this and if I were `s3-tar` developer, I'd solve the problem in the library itself. But since we can't do that...

What do you think?

Reply:
I don't love changing ENV values on the fly (it's definitely frowned upon) but this works, and I agree it's better than adding another user-defined value that is just identical in value to one we already have.